### DISCUSSION TITLE
Can I run this tool in the cloud or a VPS?

### DISCUSSION THREAD

**Discussion by Grillekkj** [KevinBytesTheDust/CVAmp#155] [Q&A]
How to optimize a azure vps for this project

**Comment by jlplenio** [comment]
You should modify the code to run without a GUI. Otherwise, you will need a super expensive Windows VPS.

**Reply by Grillekkj** [reply]
how can i do that? i tried running on a azure Basic vps and it won't work

**Reply by Bamcreek** [reply]
Did you ever figure this out?
